### PR TITLE
docs(NET-990): update release notes for v0.23.0

### PR DESCRIPTION
**Pull request by Aceix**
## Checklist before requesting a review
- [ ] My changes affect only 10 files or less.
- [ ] I have performed a self-review of my code and tested it.
- [ ] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [ ] If it is a bugfix, my code is <= 200 lines.
- [ ] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [ ] Netclient & Netmaker are awesome.

### CODE REVIEW

File: release.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-- Fixed edge case with upgrading
-- Scalability issues
+- Revamp of internet gateways: hosts and clients can now use internet gateways! More info [here](https://docs.netmaker.io/pro/internet-gateways.html)
+  On community edition, internet gateways for clients can be accessed via the Remote Access tab.
+- Support for userspace wireguard. You can now run Netclient on Linux machines without the WireGuard kernel module.
+
+## What's Fixed/Improved 🛠
+
+- Remove docker Netclient rules when the container is stopped
+- Fix for ENDPOINT environment variable
```
Comment (abhishek9686) [inline release.md:12]: this fix is for docker right?

File: release.md [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+- Stability fixes
+- Deprecate Netclient GUI
```
Comment (abhishek9686) [inline release.md:14]: Deprecated

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: release.md (hunk 1 above)
Comment (Aceix) [reply release.md:12]: yes

Comment (abhishek9686) [reply release.md:12]: mention docker netclient

**Review by abhishek9686 — APPROVED** [review]
